lib/src/models/trophie.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:draw/draw.dart';
+import 'package:draw/src/base_impl.dart';
+
+// A Class representing an award or trophy
```

Comment:
These should be triple `/` for docs to be generated.

For example, this should be `/// A class representing an award or trophy`